packages/trader/src/sass/app/modules/smart-chart.scss

```diff
@@ -90,6 +124,13 @@
                 bottom: 9.5px;
             }
         }
+        &--middle {
+            bottom: 38px;
```

Comment:
Use `rem`

Reply:
This entire file is using mostly pixels, I think it might be related to smartcharts repo not having standart for rem size. Since these styles are supposed to overwrite styles in smartcharts, do you think we need to change it or keep it as consistent with other styles in this file?